generators/app/index.js

```diff
@@ -228,10 +228,19 @@ class ConsoleGenerator extends Generator {
   async writing () {
     spinner.start()
 
-    spinner.text = 'Downloading project config...'
+    spinner.text = 'Downloading project config..'
```

Comment:
I think 3 dots are the standard way to denote an ellipsis or trailing off (the actual ellipsis character only takes up one space, but it's too small, anyways, never mind that...)

Reply:
ok I will change that :) I put three because there I saw two somewhere in the app plugin